training/proteomics/metaproteomics/metaproteomics.ga

```diff
@@ -0,0 +1,836 @@
+                }, 
+                {
+                    "label": null, 
+                    "output_name": "sqlitedb", 
+                    "uuid": "01c03ee8-9345-4e28-8bc1-ac5d3900c78d"
+                }
+            ]
+        }, 
+        "10": {
+            "annotation": "", 
+            "content_id": "toolshed.g2.bx.psu.edu/repos/iuc/sqlite_to_tabular/sqlite_to_tabular/2.0.0", 
+            "errors": null, 
+            "id": 10, 
+            "input_connections": {
+                "sqlitedb": {
+                    "id": 8, 
+                    "output_name": "sqlitedb"
+                }
+            }, 
+            "inputs": [
+                {
+                    "description": "runtime parameter for tool SQLite to tabular", 
+                    "name": "sqlitedb"
+                }
+            ], 
+            "label": "GO Terms: Biological Processes", 
+            "name": "SQLite to tabular", 
+            "outputs": [
+                {
+                    "name": "query_results", 
+                    "type": "tabular"
+                }
+            ], 
+            "position": {
+                "left": 2272.453125, 
+                "top": 189.078125
+            }, 
+            "post_job_actions": {
+                "RenameDatasetActionquery_results": {
+                    "action_arguments": {
+                        "newname": "GO Terms: Biological Processes"
+                    }, 
+                    "action_type": "RenameDatasetAction", 
+                    "output_name": "query_results"
+                }
+            }, 
+            "tool_id": "toolshed.g2.bx.psu.edu/repos/iuc/sqlite_to_tabular/sqlite_to_tabular/2.0.0", 
+            "tool_shed_repository": {
+                "changeset_revision": "fe5036ccfdc3", 
+                "name": "sqlite_to_tabular", 
+                "owner": "iuc", 
+                "tool_shed": "toolshed.g2.bx.psu.edu"
+            }, 
+            "tool_state": "{\"sqlquery\": \"\\\"SELECT go.description, \\\\ncount(distinct bering_psms.sequence) as \\\\\\\"bering_peptides\\\\\\\", count(distinct bering_psms.id) as \\\\\\\"bering_psms\\\\\\\" \\\\nFROM go JOIN bering_prot_go ON go.go_id = bering_prot_go.go_reference JOIN bering_prot on bering_prot_go.id = bering_prot.id JOIN bering_psms ON bering_prot.peptide = bering_psms.sequence\\\\nWHERE go.aspect = 'biological_process'\\\\nGROUP BY go.description\\\\nORDER BY  bering_peptides desc,bering_psms desc\\\"\", \"__page__\": null, \"sqlitedb\": \"{\\\"__class__\\\": \\\"RuntimeValue\\\"}\", \"query_result\": \"{\\\"header\\\": \\\"yes\\\", \\\"header_prefix\\\": \\\"35\\\", \\\"__current_case__\\\": 0}\", \"__rerun_remap_job_id__\": null}", 
+            "tool_version": null, 
+            "type": "tool", 
+            "uuid": "9d9634a3-436e-44fc-b43d-a606d1feae7b", 
+            "workflow_outputs": [
+                {
+                    "label": null, 
```

Comment:
the label here needs to be `query_results` as you referring to it in https://github.com/usegalaxy-eu/workflow-testing/pull/34/files#diff-4e0d654ce7c7570ef760bca55ae37f52R26